.cncf-maintainers

```diff
@@ -1,22 +1,20 @@
 approvers:
-- anik120
+- asmacdo
```

Comment:
Do we actually want to revert these changes?

Reply:
Only reason I did was because the sanity check was failing due to the OWNERS_ALIASES file not being the same on the v1.26.x branch as on master right now. If we would prefer I could update the OWNERS_ALIASES file instead.

Reply:
Updated the OWNERS_ALIASES in commit https://github.com/operator-framework/operator-sdk/pull/6290/commits/2a2de06983d851066bb66c5cef32260cd32c34d7